api/src/services/meetingsService.ts

```diff
@@ -38,6 +38,7 @@ export const findMeeting = async (
         'meeting_notes.authoring_participant_id'
       )
       .where({ 'participants.meeting_id': meetingId })
+      .orderByRaw('agenda_owning_participant_id is null')
```

Comment:
What's the difference between orderByRaw and orderBy? Why is it duplicated?

Reply:
`orderByRaw` doesn't add any quotes around the column_name that we want to order the results by in the output SQL query.

One of the column names orders by whether or not its `null`.  The other mention orders by its value.

Reply:
Thanks, that makes sense.